data/json/bionics.json

```diff
@@ -595,8 +595,11 @@
         "occupied_bodyparts": [
             [ "TORSO", 10 ]
         ],
-        "act_cost": 125,
-        "description": "Inside your body is a fleet of tiny dormant robots.  Once charged from your energy banks, they will flit about your body, repairing damage and stopping bleeding."
+        "toggled": true,
+        "act_cost": 1,
+        "react_cost": 1,
+        "time": 1,
+        "description": "Inside your body is a fleet of tiny dormant robots.  While activated they will flit about your body, repairing damage and stopping bleeding at the cost of additional power."
```

Comment:
"flit" is a fitting word. But it's also a very rare/not used causally word. Maybe just using "move" would be better?